Pod/Classes/LocationManager.swift

```diff
@@ -92,39 +92,41 @@ open class LocationManager: NSObject, CLLocationManagerDelegate {
     
     fileprivate func askWith(fulfillment: @escaping AuthorizationFulfillment, rejection: (Error) -> Void) -> Void {
 
-        var rejected = false
-
-        if askForLocationServicesFulfillments.count == 0 {
-
+        if !setupRequestPermissionsStrategy(rejection: rejection) {
+            askForLocationServicesFulfillments.append(fulfillment)
+        }
+    }
+    
+    private func setupRequestPermissionsStrategy(rejection: (Error) -> Void) -> Bool {
+        
+        if askForLocationServicesFulfillments.isEmpty {
+            
             if Bundle.main.object(forInfoDictionaryKey: "NSLocationAlwaysUsageDescription") != nil {
-
+                
```

Comment:
I recommend turning off whitespace-only lines in Xcode. It makes diffs much more cleaner.

Reply:
@mkj-is Or I can add SwiftLint for this repository.

Reply:
@mkj-is [Swiftlint integration](https://github.com/thefuntasty/LocationManager/pull/9)